dockerfiles/Dockerfile.validate

```diff
@@ -0,0 +1,9 @@
+FROM debian:stretch-slim
```

Comment:
Minor: could we name this file `Dockerfile.shellcheck` or `Dockerfile.shellvalidate` so that it's clear that it's different from "go validation"